app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineStopButton/index.js

```diff
@@ -58,7 +60,7 @@ export default class PipelineStopButton extends Component {
   }
 
   stopPipeline = () => {
-    if (this.props.stopButtonLoading || this.state.disabled) {
+    if (!this.props.isLatestVersion || this.props.stopButtonLoading || this.state.disabled) {
```

Comment:
Does this mean that we can only stop latest versions' run?

Reply:
good point. Should relax this